src/System.Runtime.Loader/tests/AssemblyLoadContextTest.cs

```diff
@@ -121,5 +121,17 @@ public static void GetLoadContextTest_ValidTrustedPlatformAssembly()
 
             Assert.NotNull(context);
         }
+
+        [Fact]
+        public static void GetLoadContextTest_SystemPrivateCorelibAssembly()
+        {
+            // System.Private.Corelib is a special case
+            // `int` is defined in S.P.C
+            var asm = typeof(int).Assembly;
+            var context = AssemblyLoadContext.GetLoadContext(asm);
+
+            Assert.NotNull(context);
+            Assert.Same(context, AssemblyLoadContext.Default);
```

Comment:
Nit: the arguments here should be reversed... the first parameter is expected and the second is actual.